docs/feature_caps_word.md

```diff
@@ -4,7 +4,10 @@ It is often useful to type a single word in all capitals, for instance
 abbreviations like "QMK", or in code, identifiers like `KC_SPC`. "Caps Word" is
 a modern alternative to Caps Lock:
 
-* Letters are capitalized while active, and Caps Word automatically disables
+* While active, letters are capitalized and `-` becomes `_`. The `_` makes it easier
+  to define PROGRAM\_CONSTANTS.
```

Comment:
Perhaps "...to type constant names (e.g. PROGRAM\_CONSTANTS)" would be better wording.

Reply:
```suggestion
  to type constant names (eg 'PROGRAM\_CONSTANTS').
```